src/Tizen.NUI/src/public/Window/Window.cs

```diff
@@ -357,6 +357,8 @@ public enum ResizeDirection
 
         /// <summary>
         /// Gets or sets a window type.
+        /// Most of window type can be set to use WindowType, except for IME type.
+        /// IME type can be set to use one of NUIApplication's constrcutors.
```

Comment:
Note: IME type can be set on application construction time only.

Reply:
I see.